user/serializers.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = (
+            "id",
+            "username",
+            "email",
+            "password",
+            "is_staff",
+        )
+        read_only = ["id", "is_staff"]
```

Comment:
The `read_only` option should be `read_only_fields`. The correct syntax is `read_only_fields = ['id', 'is_staff']` to specify fields that should be read-only.